src/api/middleware/errorHandler.ts

```
@@ -0,0 +1,217 @@
+/**
+ * Express error handling middleware
+ */
+export function errorHandler(
+  err: Error | AppError,
+  req: Request,
+  res: Response,
+  next: NextFunction
+): void {
+  // If response was already sent, delegate to default Express error handler
+  if (res.headersSent) {
+    return next(err);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)